src/game/camera.c

```diff
@@ -842,3 +851,23 @@ void Camera_RefreshFromTrigger(int16_t type, int16_t *data)
         }
     }
 }
+
+void Camera_MoveManual(void)
```

Comment:
Maybe `Camera_OffsetControl`?

Reply:
Maybe? But that name is kind of nondescript to me. The functions purpose is to move the manual camera based on the player input.

Reply:
Ok I added the Spanish translation so this PR is ready except for this discussion. I named it `Camera_MoveManual` to go along with `Camera_Move` naming convention. But if you think I should switch it, I will.